configuration/testversions.properties

```diff
@@ -0,0 +1,8 @@
+2.17.0.redhat-630187=
```

Comment:
I don't see the interest of having an external files retrieved/updated from the web for versions tested.
Why not hardcoding it in a first time?
Maybe having a possibility to test against a larger range of versions later on would be interesting, but I would use a system property for that and make it configurable just by providing parameters to the maven build.

What's your pros and cons on both solutions?